lite2/client.go

```diff
@@ -965,13 +965,19 @@ func (c *Client) RemoveNoLongerTrustedHeaders(now time.Time) {
 		c.logger.Error("can't get first trusted height", "err", err)
```

Comment:
maybe update `c.trustedStore.FirstSignedHeaderHeight()` to `c.FirstTrustedHeight()`